src/fairseq2/models/conformer/convolution.py

```diff
@@ -42,6 +46,9 @@ def __init__(
             The dimensionality of the model.
         :param depthwise_kernel_size:
             The kernel size of the depthwise convolution.
+        :param causal_depthwise_conv:
+            If True, uses a causal depthwise convolution similar to that described in
+            Section 2.1 of :cite:t:`https://doi.org/10.48550/arxiv.2006.11477`.
```

Comment:
This is wav2vec 2.0 paper right? I don't see a Section 2.1 or a reference to causality there. Are you sure that this is the right reference. Also I forgot to mention it in the first review, but you should also put the bibtex of the paper to bibliography.bib at the root of the repository.

Reply:
Oh no, wrong link sorry!